modules/wp-cli/library.php

```diff
@@ -77,12 +81,14 @@ public function sync( $args, $assoc_args ) {
 	 * @since  2.8.0
 	 * @access public
 	 */
-	public function import( $args ) {
+	public function import( $args, $assoc_args ) {
 		if ( empty( $args[0] ) ) {
 			\WP_CLI::error( 'Please set file path.' );
 		}
 
 		$file = $args[0];
+		$imported_items_ids = [];
+		$return_type = \WP_CLI\Utils\get_flag_value( $assoc_args, 'returnType', 'info' );
```

Comment:
Can you get it directly from `assoc_args`?

Reply:
I can do it with `$assoc_args['returnType']`, but it was the quickest way the get the value and set a default value if not exist.